src/features/surveys/components/SurveySubmissionsList.tsx

```diff
@@ -157,20 +161,28 @@ const SurveySubmissionsList = ({
     }
   };
 
-  const EditCell: FC<{ row: ZetkinSurveySubmission }> = ({ row }) => {
+  const EditCell: FC<{
+    respondents: ZetkinPerson[];
+    row: ZetkinSurveySubmission;
+  }> = ({ respondents, row }) => {
     const api = useGridApiContext();
     const { orgId } = useRouter().query;
-    const email = row.respondent?.email || '';
+
+    const emailOrName =
+      row.respondent?.email ||
+      row.respondent?.first_name ||
+      row.respondent?.last_name ||
+      '';
     let { data: suggestedPeople } = useQuery(
-      ['peopleSearchResults', email],
-      getPeopleSearchResults(email, orgId as string),
+      ['peopleSearchResults', emailOrName],
+      getPeopleSearchResults(emailOrName, orgId as string),
       {
```

Comment:
I assum what's happening here is that I'm getting three "Casey" because they get included one time for each search criteria (first name, last name and email)? So iI think this needs to be filtered down to avoid duplicates
![bild](https://user-images.githubusercontent.com/58265097/227642074-63e955a7-c958-46f8-b6e1-61fa223f02b8.png)

Reply:
Good catch! I´m filtering now to avoid duplicates :D